msteams-platform/sbs-teams-app-test-tool.yml

```diff
@@ -0,0 +1,351 @@
+
+    1. Select **Create a New App**.
+
+        :::image type="content" source="~/assets/images/teams-toolkit-v2/create-project.png" alt-text="Screenshot shows the location of the Create New Project link in the Teams Toolkit sidebar.":::
+
+    1. Select **Bot**.
+
+        :::image type="content" source="~/assets/images/teams-toolkit-v2/first-bot/create-new-app1.png" alt-text="Screenshot shows the Teams Toolkit app templates.":::
+
+    1. Select **AI Chat Bot**. If you need a different functionality for your bot, select another option.
+
+        :::image type="content" source="~/assets/images/teams-toolkit-v2/debug/ai-chat-bot.png" alt-text="Screenshot shows the app feature to add to your new app.":::
+
+    1. Select **JavaScript**.
+
+        :::image type="content" source="~/assets/images/test-tool/select-language-bot.png" alt-text="Screenshot shows the option to select the programming language.":::
+
+    1. Select **Default folder**.
+
+        :::image type="content" source="~/assets/images/teams-toolkit-v2/first-bot/select-default-location.png" alt-text="Screenshot shows the selection of default location.":::
+
+        To change the default location, follow these steps:
+
+        1. Select **Browse**.
+
+            :::image type="content" source="~/assets/images/teams-toolkit-v2/first-bot/select-browse.png" alt-text="Screenshot shows the selection of browse location option.":::
+
+        1. Select the location for the project workspace.
+        1. Select **Select Folder**.
+
+            :::image type="content" source="~/assets/images/teams-toolkit-v2/select-folder.png" alt-text="Screenshot shows the folder to select.":::
+
+        1. Enter a suitable name for your app and then select the **Enter** key.
+
+            :::image type="content" source="~/assets/images/teams-toolkit-v2/first-bot/hello-bot.png" alt-text="Screenshot shows where to enter the app name.":::
+
+            A dialog appears, where you need to choose yes or no to trust the authors of the files in this folder.
+
+            :::image type="content" source="~/assets/images/teams-toolkit-v2/first-bot/vsc-trust-authors.png" alt-text="Screenshot shows the dialog to trust or not the authors of the files in this folder.":::
+
+            Now you've successfully created your AI chat bot project workspace.
+
+    ## Take a tour of the bot app source code
+
+    After you finish scaffolding, explore the project directories and files in the Explorer section of the Visual Studio Code.
+    
+    :::image type="content" source="../msteams-platform/assets/images/test-tool/source-code.png" alt-text="Screenshot shows the Teams Toolkit sample bot folder Structure.":::
+    
+    | Folder name | Contents |
+    | --- | --- |
+    | `env/.env.testtool.user` |Configuration file for local environment used by `teamsapp.testtool.yml` to customize the provisioning and deployment rules.  |
```

Comment:
the env file name for test tool should be ``env/.env.testtool`

Reply:
Updated in the table as suggested.